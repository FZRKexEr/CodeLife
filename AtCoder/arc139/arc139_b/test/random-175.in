100
30 11 18 10 13 12
60 10 8 1 14 4
14 20 19 11 13 18
58 6 6 3 20 9
89 10 7 5 14 17
100 16 3 1 18 19
5 12 4 4 15 2
80 7 4 10 3 2
42 4 18 13 18 10
7 20 3 16 4 6
68 20 3 8 20 1
17 3 13 9 12 7
92 19 4 3 12 16
15 6 8 11 6 17
54 18 14 17 12 12
84 5 7 2 11 4
76 3 16 6 20 2
81 9 8 7 3 5
66 11 5 18 12 2
93 19 3 1 19 8
25 10 9 10 17 2
70 7 16 3 7 10
9 8 11 4 5 14
10 14 13 14 16 1
99 15 18 13 14 16
86 16 17 18 19 20
58 16 18 4 10 6
79 12 10 12 9 10
38 15 20 6 7 16
2 20 19 5 11 4
10 18 2 18 13 5
57 8 4 4 20 19
57 7 18 18 17 12
40 14 20 11 7 1
60 4 3 12 10 13
72 14 7 3 20 6
31 6 20 16 4 13
53 5 19 17 20 20
90 20 8 13 12 16
12 14 3 13 6 14
91 8 5 3 1 17
38 4 14 17 4 17
76 15 14 2 4 8
68 15 18 8 6 9
82 4 10 11 17 20
65 19 19 3 19 15
27 20 15 1 11 19
74 2 11 12 11 14
86 2 10 19 7 19
59 1 17 3 12 12
26 10 6 7 9 4
35 11 12 17 15 14
92 4 18 10 18 5
21 4 3 6 10 6
32 7 1 9 7 4
55 6 17 19 3 9
91 6 5 20 2 11
22 3 20 6 10 14
56 18 2 13 20 11
2 20 19 1 7 3
99 20 6 12 10 8
2 19 5 6 5 5
20 2 1 20 10 14
4 7 7 4 4 10
16 8 4 16 5 14
14 18 2 9 16 17
76 1 8 14 13 6
46 19 12 4 9 16
99 8 14 11 9 2
77 20 9 2 16 12
46 9 5 2 6 3
26 18 7 18 18 11
4 5 1 6 10 10
50 16 2 16 9 10
52 12 4 4 20 9
14 5 3 11 11 5
86 1 7 5 16 11
54 3 7 14 18 19
58 18 9 5 2 18
76 19 17 7 17 4
6 16 15 14 18 17
50 19 3 1 2 19
26 18 19 20 17 14
83 9 14 6 13 7
48 8 18 20 8 7
39 5 16 3 12 14
15 7 1 2 15 4
28 18 16 13 16 12
13 9 11 2 2 6
42 1 7 18 18 20
72 6 5 4 19 19
15 17 17 1 2 19
23 4 2 12 1 2
30 17 17 14 5 3
14 8 6 7 2 18
94 15 8 11 15 17
74 7 19 12 15 2
67 8 7 20 3 14
28 19 19 8 16 8
75 19 2 2 12 7
