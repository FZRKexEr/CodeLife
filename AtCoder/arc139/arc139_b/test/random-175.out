35
32
154
93
139
100
6
36
64
28
36
28
96
34
36
48
80
27
34
93
74
54
9
140
90
118
34
78
41
10
25
144
75
2
150
66
36
100
90
56
20
41
22
46
182
69
18
128
173
60
22
43
45
42
44
46
50
26
71
2
76
12
100
16
10
119
106
50
25
76
29
47
24
37
89
26
142
179
26
68
84
16
154
67
48
46
30
146
10
120
228
15
19
185
20
134
72
82
80
72
